bin/start_be.sh

```diff
@@ -16,11 +16,38 @@
 # specific language governing permissions and limitations
 # under the License.
 
-curdir=$(dirname "$0")
-curdir=$(
-    cd "$curdir"
-    pwd
-)
+#check max_map_count
+MAX_MAP_COUNT=`sysctl -n vm.max_map_count`
```

Comment:
```suggestion
MAX_MAP_COUNT="$(sysctl -n vm.max_map_count)"
```suggestion

It is better to use `$()` rather than ``.